Scheduler Customize Individual Views: rewrite (#912)
DevExpress/devextreme-demos: JSDemos/Demos/Scheduler/CustomizeIndividualViews/description.md

> To customize a view, configure its settings in an object of the [views[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views) array. You should specify the view's [type](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type) and other properties that override global Scheduler settings.

Reviewer: An object should be an element of the **views[]** array that means this object is **in** the array, right?

```suggestion
To customize a view, configure its settings in an object in the [views[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views) array. You should specify the view's [type](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type) and other properties that override global Scheduler settings.
```

> To customize a view, configure its settings in an object of the [views[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views) array. You should specify the view's [type](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type) and other properties that override global Scheduler settings.

Reviewer 2: ```suggestion
To customize a view, configure its settings in an object of the [views[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views) array. You should specify the view's [type](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type) and other properties to override global view settings.
```

> This demo has two customized views: *"week"* and *"workWeek"*. They group appointments by resources and specify the [dateCellTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#dateCellTemplate). In addition, the *"workWeek"* view specifies the [startDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#endDayHour) values.

Reviewer 2: ```suggestion
This demo shows four views, two of which (Week and Work Week) are customized. Both customized views [group appointments](/Demos/WidgetsGallery/Demo/Scheduler/GroupOrientation/) by [resources](/Demos/WidgetsGallery/Demo/Scheduler/Resources/) and use the [dateCellTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#dateCellTemplate) to change the appearance of date cells. In addition, the [startDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#endDayHour) properties are used to narrow down the time window on the Work Week view.
```

> This demo shows four views, two of which (Week and Work Week) are customized. Both customized views [group appointments](/Demos/WidgetsGallery/Demo/Scheduler/GroupOrientation/) by [resources](/Demos/WidgetsGallery/Demo/Scheduler/Resources/) and use the [dateCellTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#dateCellTemplate) to change the appearance of date cells. In addition, the [startDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#endDayHour) properties are used to narrow down the time window on the Work Week view.

Reviewer 3: `... two of which ...` - sign of a complicated sentence. You can even put a period after `four views` and start the next sentence with `Week and Work Week views`. 

`are customized` - passive. 

'properties are used' - passive. 

The list of changes is written as a narrative, where I would suggest a more structured approach (a list). 

So here's the suggestion that fixes all those comments. 

```suggestion
This demo customizes two views - Week and Work Week - in the following manner: 

* Both views [group appointments](/Demos/WidgetsGallery/Demo/Scheduler/GroupOrientation/) by [resources](/Demos/WidgetsGallery/Demo/Scheduler/Resources/).
* Both views use the [dateCellTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#dateCellTemplate) to change the appearance of date cells.
* The Work Week view changes the [startDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#endDayHour) properties to limit the time range.

Day and Month views use default settings.
```

> To add a customized view, create an object in the [views[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views) array. In this object, you should specify the view's [type](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type) and other properties that will override global Scheduler configuration.

Reviewer: We can be much more straightforward here:

```suggestion
To customize a view, specify its settings in an object in the [views[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views) array. You should specify the view's [type](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type) and other properties that override global Scheduler configuration.
```

> This demo has two customized views: *"week"* and *"workWeek"*. What differs them
> is that they group appointments by resources and set the [dateCellTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#dateCellTemplate). In addition, the *"workWeek"* view specifies the [startDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#endDayHour) values.

Reviewer: ```suggestion
This demo has two customized views: *"week"* and *"workWeek"*. They group appointments by resources and specify the [dateCellTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#dateCellTemplate). In addition, the *"workWeek"* view specifies the [startDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#endDayHour) values.
```
